### PR TITLE
migrate to jhipster 6.5.1

### PR DESCRIPTION
**Pull request by yevheniimuzyka**

### CODE REVIEW

File: src/main/docker/entrypoint.sh [hunk 1]
```diff
@@ -1,30 +1,4 @@
-#!/bin/bash
-set -e
```
Comment (vkirichenko) [inline src/main/docker/entrypoint.sh]: why this file was removed

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
+#Thu Dec 26 18:06:28 EET 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```
Comment (vkirichenko) [inline gradle/wrapper/gradle-wrapper.properties]: gradle-6.0.1-bin.zip should be used

File: src/main/java/com/icthh/xm/tmf/ms/resource/ResourceApp.java [hunk 3]
```diff
@@ -19,6 +14,12 @@
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 import org.springframework.core.env.Environment;
 
+import javax.annotation.PostConstruct;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Arrays;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/ResourceApp.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/aop/logging/LoggingAspect.java [hunk 4]
```diff
@@ -12,6 +11,8 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.core.env.Environment;
 
+import java.util.Arrays;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/aop/logging/LoggingAspect.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/client/AuthorizedFeignClient.java [hunk 5]
```diff
@@ -1,14 +1,11 @@
...
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.cloud.openfeign.FeignClientsConfiguration;
 import org.springframework.core.annotation.AliasFor;
 
+import java.lang.annotation.*;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/client/AuthorizedFeignClient.java]: wildcard imports should be removed, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/client/AuthorizedUserFeignClient.java [hunk 6]
```diff
@@ -1,14 +1,11 @@
...
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.cloud.openfeign.FeignClientsConfiguration;
 import org.springframework.core.annotation.AliasFor;
 
+import java.lang.annotation.*;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/client/AuthorizedUserFeignClient.java]: wildcard imports should be removed, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2InterceptedFeignConfiguration.java [hunk 7]
```diff
@@ -2,11 +2,12 @@
 
 import feign.RequestInterceptor;
 import io.github.jhipster.security.uaa.LoadBalancedResourceDetails;
-import java.io.IOException;
 import org.springframework.cloud.security.oauth2.client.feign.OAuth2FeignRequestInterceptor;
 import org.springframework.context.annotation.Bean;
 import org.springframework.security.oauth2.client.DefaultOAuth2ClientContext;
 
+import java.io.IOException;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2InterceptedFeignConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2UserClientFeignConfiguration.java [hunk 8]
```diff
@@ -1,9 +1,10 @@
 package com.icthh.xm.tmf.ms.resource.client;
 
 import feign.RequestInterceptor;
-import java.io.IOException;
 import org.springframework.context.annotation.Bean;
 
+import java.io.IOException;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2UserClientFeignConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/AppMetricsConfiguration.java [hunk 9]
```diff
@@ -4,12 +4,13 @@
 import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
 import com.ryantenney.metrics.spring.config.annotation.MetricsConfigurerAdapter;
 import com.zaxxer.hikari.HikariDataSource;
-import javax.annotation.PostConstruct;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
 
+import javax.annotation.PostConstruct;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/AppMetricsConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/ApplicationProperties.java [hunk 10]
```diff
@@ -1,13 +1,14 @@
...
 
 import com.icthh.xm.commons.lep.TenantScriptStorage;
-import java.util.Collections;
-import java.util.List;
 import lombok.Data;
 import lombok.Getter;
 import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.Collections;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/ApplicationProperties.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/AsyncConfiguration.java [hunk 11]
```diff
@@ -17,6 +15,9 @@
 import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
 import org.springframework.scheduling.config.ScheduledTaskRegistrar;
 
+import java.util.concurrent.Executor;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/AsyncConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/AsyncConfiguration.java [hunk 12]
```diff
@@ -46,7 +47,7 @@ public Executor getAsyncExecutor() {
     public AsyncUncaughtExceptionHandler getAsyncUncaughtExceptionHandler() {
         return new SimpleAsyncUncaughtExceptionHandler();
     }
-    
+
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/AsyncConfiguration.java]: Unnecessary changes

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/CloudDatabaseConfiguration.java [hunk 13]
```diff
@@ -10,13 +9,15 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
+import javax.sql.DataSource;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/CloudDatabaseConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/DatabaseConfiguration.java [hunk 14]
```diff
@@ -13,6 +12,8 @@
 import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
+import java.sql.SQLException;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/DatabaseConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/DefaultProfileUtil.java [hunk 15]
```diff
@@ -1,11 +1,12 @@
 package com.icthh.xm.tmf.ms.resource.config;
 
 import io.github.jhipster.config.JHipsterConstants;
-import java.util.HashMap;
-import java.util.Map;
 import org.springframework.boot.SpringApplication;
 import org.springframework.core.env.Environment;
 
+import java.util.HashMap;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/DefaultProfileUtil.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/DefaultProfileUtil.java [hunk 16]
```diff
@@ -26,10 +27,10 @@ private DefaultProfileUtil() {
...
         Map<String, Object> defProperties = new HashMap<>();
         /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
+         * The default profile to use when no other profiles are defined
+         * This cannot be set in the <code>application.yml</code> file.
+         * See https://github.com/spring-projects/spring-boot/issues/1219
+         */
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/DefaultProfileUtil.java]: Unnecessary changes

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/WebConfigurer.java [hunk 17]
```diff
@@ -31,6 +25,10 @@
 import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
 import org.springframework.web.filter.CorsFilter;
 
+import javax.servlet.*;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/WebConfigurer.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

+ wildcard imports

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/WebMvcConfiguration.java [hunk 18]
```diff
@@ -4,21 +4,22 @@
 import com.icthh.xm.commons.web.spring.TenantInterceptor;
 import com.icthh.xm.commons.web.spring.XmLoggingInterceptor;
 import com.icthh.xm.commons.web.spring.config.XmWebMvcConfigurerAdapter;
-import java.util.List;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
 
+import java.util.List;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/WebMvcConfiguration.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/config/audit/AuditEventConverter.java [hunk 19]
```diff
@@ -1,16 +1,12 @@
...
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
 import org.springframework.boot.actuate.audit.AuditEvent;
 import org.springframework.security.web.authentication.WebAuthenticationDetails;
 import org.springframework.stereotype.Component;
 
+import java.util.*;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/config/audit/AuditEventConverter.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/domain/AbstractAuditingEntity.java [hunk 20]
```diff
@@ -1,18 +1,19 @@
...
 import org.hibernate.envers.Audited;
 import org.springframework.data.annotation.CreatedBy;
 import org.springframework.data.annotation.CreatedDate;
 import org.springframework.data.annotation.LastModifiedBy;
 import org.springframework.data.annotation.LastModifiedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/domain/AbstractAuditingEntity.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/domain/PersistentAuditEvent.java [hunk 21]
```diff
@@ -1,22 +1,22 @@
...
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.MapKeyColumn;
+import javax.persistence.CollectionTable;
+import javax.persistence.JoinColumn;
 import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/domain/PersistentAuditEvent.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/lep/XmMsLepProcessingApplicationListener.java [hunk 22]
```diff
@@ -1,23 +1,19 @@
...
-import java.util.HashMap;
-import java.util.Map;
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.*;
+
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/lep/XmMsLepProcessingApplicationListener.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/security/SecurityUtils.java [hunk 23]
```diff
@@ -1,10 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.security;
 
-import java.util.Optional;
 import org.springframework.security.core.context.SecurityContext;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.core.userdetails.UserDetails;
 
+import java.util.Optional;
+
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/security/SecurityUtils.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/security/SpringSecurityAuditorAware.java [hunk 24]
```diff
@@ -1,10 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.security;
 
 import com.icthh.xm.tmf.ms.resource.config.Constants;
-import java.util.Optional;
 import org.springframework.data.domain.AuditorAware;
 import org.springframework.stereotype.Component;
 
+import java.util.Optional;
+
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/security/SpringSecurityAuditorAware.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/web/rest/errors/BadRequestAlertException.java [hunk 25]
```diff
@@ -1,10 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.web.rest.errors;
 
+import org.zalando.problem.AbstractThrowableProblem;
+import org.zalando.problem.Status;
+
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/web/rest/errors/BadRequestAlertException.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/java/com/icthh/xm/tmf/ms/resource/web/rest/errors/ExceptionTranslator.java [hunk 26]
```diff
@@ -19,14 +13,22 @@
...
 import org.zalando.problem.spring.web.advice.ProblemHandling;
+import org.zalando.problem.spring.web.advice.security.SecurityAdviceTrait;
 import org.zalando.problem.violations.ConstraintViolationProblem;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
```
Comment (vkirichenko) [inline src/main/java/com/icthh/xm/tmf/ms/resource/web/rest/errors/ExceptionTranslator.java]: Unnecessary changes, according checkstyle imports orders must be the same as in old version

File: src/main/docker/app.yml [hunk 27]
```diff
@@ -1,35 +1,25 @@
 version: '2'
 services:
-    resource-app:
```
Comment (amedvedchuk) [inline src/main/docker/app.yml]: @yevheniimuzyka 
why TAB width was changed from 4 to 2 spaces? it produces a lot of changes.
